tests/chgres_cube/ftst_search_util.F90

```diff
@@ -0,0 +1,186 @@
+ program test_search_util
+
+! Test search_util using a simple 3x3 fv3 tile. Two types
+! of tests are performed. First, a missing value is 
+! replaced with a valid neighboring value. Second, a missing
+! value is replaced by a default value. This can happen
+! for an isolated island located far from a valid neighbor.
```

Comment:
Excellent documentation here and throughout. It makes everything much more clear!

Please add your name here. Although we don't use doxygen, all tests should still include their author.